plugins/database/quota/src/org/apache/cloudstack/api/response/QuotaResponseBuilderImpl.java

```diff
@@ -0,0 +1,419 @@
+            dummy.setQuotaUsed(new BigDecimal(0));
+            quotaUsage.add(dummy);
+        }
+
+        Collections.sort(quotaUsage, new Comparator<QuotaUsageVO>() {
+            public int compare(QuotaUsageVO o1, QuotaUsageVO o2) {
+                if (o1.getUsageType() == o2.getUsageType())
+                    return 0;
+                return o1.getUsageType() < o2.getUsageType() ? -1 : 1;
+            }
+        });
+
+        List<QuotaStatementItemResponse> items = new ArrayList<QuotaStatementItemResponse>();
+        QuotaStatementItemResponse lineitem;
+        int type = -1;
+        BigDecimal usage = new BigDecimal(0);
+        BigDecimal totalUsage = new BigDecimal(0);
+        quotaUsage.add(new QuotaUsageVO());// boundary
+        QuotaUsageVO prev = quotaUsage.get(0);
+        // s_logger.info("createQuotaStatementResponse record count=" + quotaUsage.size());
+        for (final QuotaUsageVO quotaRecord : quotaUsage) {
+            // s_logger.info("createQuotaStatementResponse Type=" + quotaRecord.getUsageType() + " usage=" + usage + " name" + quotaRecord.getUsageItemId());
+            if (type != quotaRecord.getUsageType()) {
+                if (type != -1) {
+                    lineitem = new QuotaStatementItemResponse();
+                    lineitem.setUsageType(type);
+                    lineitem.setQuotaUsed(usage);
+                    lineitem.setAccountId(prev.getAccountId());
+                    lineitem.setDomainId(prev.getDomainId());
+                    lineitem.setStartDate(prev.getStartDate());
+                    lineitem.setEndDate(prev.getEndDate());
+                    lineitem.setUsageUnit(quotaTariffMap.get(type).getUsageUnit());
+                    lineitem.setUsageName(quotaTariffMap.get(type).getUsageName());
+                    lineitem.setObjectName("quotausage");
+                    items.add(lineitem);
+                    totalUsage = totalUsage.add(usage);
+                    usage = new BigDecimal(0);
+                }
+                type = quotaRecord.getUsageType();
+            }
+            prev = quotaRecord;
+            usage = usage.add(quotaRecord.getQuotaUsed());
+        }
+        TransactionLegacy.open(opendb).close();
+
+        statement.setLineItem(items);
+        statement.setTotalQuota(totalUsage);
+        statement.setCurrency(QuotaConfig.QuotaCurrencySymbol.value());
+        statement.setObjectName("statement");
+        return statement;
+    }
+
+    @Override
+    public List<QuotaTariffVO> listQuotaTariffPlans(final QuotaTariffListCmd cmd) {
+        List<QuotaTariffVO> result = new ArrayList<QuotaTariffVO>();
+        Date effectiveDate = cmd.getEffectiveDate() == null ? new Date() : cmd.getEffectiveDate();
+        Date adjustedEffectiveDate = _quotaService.computeAdjustedTime(effectiveDate);
+        s_logger.info("Effective datec=" + effectiveDate + " quotatype=" + cmd.getUsageType() + " Adjusted date=" + adjustedEffectiveDate);
```

Comment:
Logging to INFO should provide contextual information useable by system administrators.  This message seems more appropriate as DEBUG or it needs more context to be useful by a system administrator.